bezier-rs/lib/src/subpath/core.rs

```diff
@@ -0,0 +1,97 @@
+use super::*;
+
+/// Functionality relating to core `SubPath` operations, such as constructors and `iter`.
+impl SubPath {
```

Comment:
Rename `SubPath` to `Subpath` everywhere including the JS part of the code (search the project and enable capitalization checking).